File: src/consumer/EventTriangleAPI.Consumer.BusinessLogic/Models/UserDto.cs
using EventTriangleAPI.Shared.DTO.Enums;

namespace EventTriangleAPI.Consumer.BusinessLogic.Models;

public record UserDto(
    string Id,
    string Email, 
    UserRole UserRole,
    UserStatus UserStatus,
    Guid WalletId,
    WalletDto Wallet);

File: src/authorization/EventTriangle.Client/src/app/types/models/UserDto.ts
import {UserRole} from "../enums/UserRole";
import {UserStatus} from "../enums/UserStatus";
import {WalletDto} from "./WalletDto";

export interface UserDto {
  id: string;
  email: string;
  userRole: UserRole;
  userStatus: UserStatus;
  walletId: string;
  wallet: WalletDto;
}
